src/vivarium_cluster_tools/psimulate/worker/vivarium_work_horse.py

```diff
@@ -56,68 +148,16 @@ def work_horse(job_parameters: dict) -> tuple[pd.DataFrame, dict[str, pd.DataFra
         logger.info("Beginning simulation setup.")
         backup = get_backup(job_parameters)
         if backup:
-            logger.info(f"Restarting simulation from saved backup")
-            sim = backup
-            event["simulation_start"] = time()
-            exec_time = {
-                "setup_minutes": (event["simulation_start"] - event["start"]) / 60
-            }  # execution event
-            logger.info(
-                f'Simulation setup completed in {exec_time["setup_minutes"]:.3f} minutes.'
-            )
+            get_sim_from_backup(event, backup)
```

Comment:
This can't be working; you're not returning `sim` or `exec_time`.

Reply:
This also implies that we have no test coverage here. Do we not have tests that stress the backup functionality? Can you maybe add a ticket to add some?